pkg/virt-handler/notify-server/server.go

```diff
@@ -156,8 +156,7 @@ func RunServer(virtShareDir string, stopChan chan struct{}, c chan watch.Event,
 	case <-stopChan:
 		log.Log.Info("stopping notify server")
 		grpcServer.Stop()
-		sock.Close()
-		os.Remove(sockFile)
+		log.Log.Info("notify server stopped")
```

Comment:
i would suggest to add a print also before line 149,
(i know it wont be atomic)
the reason is that the server is a singleton per virt-handler, (and is recreated if its dies).
We have a race that a go routine of the server isnt closed (or is using it before its ready),
so it would help to know generally what is the flow upon server creation and closing.